community/kernel/src/main/java/org/neo4j/helpers/Settings.java

```diff
@@ -830,6 +832,23 @@ public String apply( Function<String, String> settings )
         };
     }
 
+    public static double getDouble(String propertyName, double defaultValue)
```

Comment:
I think this fits better in [FeatureToggles](https://github.com/neo4j/neo4j/blob/3.0/community/unsafe/src/main/java/org/neo4j/unsafe/impl/internal/dragons/FeatureToggles.java).